frontend/packages/employee-frontend/src/components/Reports.tsx

```diff
@@ -76,7 +76,7 @@ function Reports() {
               <TitleRow>
                 <RoundIcon
                   size="L"
-                  color={EspooColours.orange}
+                  color={colors.accents.orange}
```

Comment:
I would suggest that the color for round icon would be defined something like `{color.roundicom}`, because it would tell a lot more to the person that is configuring the colors to a city specific instance of evaka. The colour could be something else than orange, right?

Reply:
Yes, actually I wrote this in slack:

> Huomenna vois puhua vähän evakan värikartasta teemoituksen näkökulmasta. Puolikuvitteellinen huonohko esimerkki: Nyt väri voi olla nimetty vaikkapa orange ja sitten sitä käytetään vaikkapa sekä warning-messagen värityksessä että varahoito-labelin värinä. Tästä seuraa kaksi ongelmaa: 1) Jos tampere haluaakin käyttää varoitusvärinä oranssin sijaan keltaista niin on hämäävää kun värin nimi on edelleen oranssi. 2) Jos tampere haluaa muuttaa varahoitolabelin vihreäksi niin samalla varoitustekstitkin muuttuu vihreiksi. Etenkin ongelmia tulee jos jossain on liikaa luotettu siihen et primary color sattuu olemaan sininen, vaikkapa linkeissä, ja sit kun tampere muuttaakin primary colorin vaikka oranssiksi niin linkit muuttuu samalla.
> 
> Eli värit tarviis nimetä enempi käyttötarkoituksen mukaan eikä sen mukaan minkä värisiä ne sattuu nyt espoon teemassa olemaan.
> 
> Hyviä nimiä: success, warning, error, info, link
> Huonoja nimiä: water, limeGreen, primaryBlue
> 
> Tää voi olla liian iso remontti korjata kokonaan, mut voitais vähän miettiä mitkä ois tärkeitä ja helppoja paloja tuosta.

But yeah, need to think what is worth fixing at this point. Renaming is easy, but some should ideally be split into multiple variables and that would require going through every single usage manually, potentially hundreds, and deciding which one is correct. So maybe we'll wait for some input on what things actually even need customizing and then focus on making those work.

PS: I'll put this PR back into draft state since the idea we had for the build system did not quite work out after all, so the whole  approach may need to be thought out once more.